core/benchmarks/IteratorSizeBenchmark-results.txt

```diff
@@ -0,0 +1,105 @@
+OpenJDK 64-Bit Server VM 1.8.0_342-b07 on Linux 5.15.0-1014-azure
```

Comment:
https://github.com/apache/spark/blob/82cd993244656153551c9bb122247789d2a5b6b4/core/src/main/scala/org/apache/spark/util/Utils.scala#L1910-L1915

From the benchmark results, the above conclusion is correct only when `Scala 2.12 + Java 8` is used.